src/main/webapp/js/build.js

```diff
@@ -5,7 +5,13 @@ if (rebuildButton) {
     event.preventDefault();
     const buildUrl = `${rebuildButton.dataset.buildPath}?delay=0sec`
     if (rebuildButton.dataset.parameterized === 'true') {
-      window.location.href = buildUrl
+      var rebuildAction = window[`${rebuildButton.dataset.proxyName}`];
+      rebuildAction.doRebuildjob(function (success) {
+        var result = success.responseJSON;
+        if (result) {
+          window.hoverNotification(rebuildButton.dataset.successMessage, rebuildButton);
```

Comment:
as a future enhancement, would be good to provide a link to the scheduled build.

Reply:
@timja you mean in the popup? I need even to think to show an error when you can't rebuild. Basically when you have a field that has a password you can't rebuild the job. I suggest to have two popup, one in green Build scheduled and one in red.